src/devices/bus/heathzenith/h8/h8bus.h

```diff
@@ -0,0 +1,412 @@
+    +8 V           48,49                       +8 V Unregulated
+    -18 V          2                          -18 V Unregulated
+    GND            0,1                        Ground
+    CPU GND        18,24                      CPU Ground
+
+
+***************************************************************************
+
+  P201 Cable
+
+  Between P1 and P2 was a cable labeled P201 in the schematics. This connected
+  the Front Panel(P1) card to the CPU board(P2), providing dedicated signal lines
+  between the two boards.
+
+    Signal         Pin       Direction        Description
+    --------------------------------------------------------
+
+    GND             1                         Ground
+    /INT2 (20)      2        P1 -> P2         Interrupt 20
+    /INTE           3        P2 -> P1         Interrupts enabled.
+    /INT1 (10)      4        P1 -> P2         Interrupt 10
+    /RESIN          5        P1 -> P2         Reset in, allowed keypad to initiate a reset of the computer
+
+***************************************************************************/
+
+#ifndef MAME_BUS_HEATHZENITH_H8_H8BUS_H
+#define MAME_BUS_HEATHZENITH_H8_H8BUS_H
+
+#pragma once
+#include <functional>
+#include <utility>
+#include <vector>
+
+
+class h8bus_device;
+
+
+class device_h8bus_card_interface : public device_interface
+{
+	friend class h8bus_device;
+public:
+	virtual ~device_h8bus_card_interface();
+
+	// signals from the Bus
+	virtual void int1_w(int state) {}
+	virtual void int2_w(int state) {}
+	virtual void int3_w(int state) {}
+	virtual void int4_w(int state) {}
+	virtual void int5_w(int state) {}
+	virtual void int6_w(int state) {}
+	virtual void int7_w(int state) {}
+
+	virtual void m1_w(int state) {}
+	virtual void reset_w(int state) {}
+	virtual void hold_w(int state) {}
+	virtual void rom_disable_w(int state) {}
+
+	void set_h8bus_tag(h8bus_device *h8bus, const char *slottag) { m_h8bus = h8bus; m_h8bus_slottag = slottag; }
+	void set_slot_num(u8 slot_num) { m_slot_num = slot_num; }
+
+protected:
+	device_h8bus_card_interface(const machine_config &mconfig, device_t &device);
+	virtual void interface_pre_start() override;
+
+	void set_slot_int1(int state);
+	void set_slot_int2(int state);
+	void set_slot_int3(int state);
+	void set_slot_int4(int state);
+	void set_slot_int5(int state);
+	void set_slot_int6(int state);
+	void set_slot_int7(int state);
+	void set_slot_reset(int state);
+	void set_slot_hold(int state);
+	void set_slot_rom_disable(int state);
+	void set_slot_m1(int state);
+
+	h8bus_device &h8bus() { assert(m_h8bus); return *m_h8bus; }
+
+	const char *m_h8bus_slottag;
+	u8 m_slot_num;
+
+private:
+	h8bus_device *m_h8bus;
+};
+
+class device_h8bus_p1_card_interface : public device_h8bus_card_interface
+{
+public:
+	virtual ~device_h8bus_p1_card_interface();
+
+	virtual void p201_inte(int state) {}
+
+protected:
+	device_h8bus_p1_card_interface(const machine_config &mconfig, device_t &device);
+
+	void set_p201_reset(int state);
```

Comment:
Is it possible to use a finder in the driver to locate the p1 and p2 cards and hook these signals card-to-card instead of via the bus?